src/Acme.UI/Infrastructure/Handlers/PublishHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Umbraco.Core.Composing;
+using Umbraco.Core.Events;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Services;
+using Umbraco.Core.Services.Implement;
+
+namespace Acme.UI.Infrastructure.Handlers
+{
+    [RuntimeLevel(MinLevel = Umbraco.Core.RuntimeLevel.Run)]
+    public class PublishHandlerComposer : ComponentComposer<PublishHandler>
+    {
+    }
+
+    public class PublishHandler : IComponent
+    {
+        private ILogger _logger;
+
+        public PublishHandler(ILogger logger)
+        {
+            _logger = logger;
+        }
+
+        public void Initialize()
+        {
+            // ContentService.Published += this.OnPublish;
```

Comment:
Remove commented code please

Reply:
We need the subscription code here so un-commented the code. We needed for OnPublish call back to happen on publish action.